recipe/build.sh

```diff
@@ -28,6 +28,7 @@ USED_BUILD_PREFIX=${BUILD_PREFIX:-${PREFIX}}
 echo USED_BUILD_PREFIX=${BUILD_PREFIX}
 
 MAKE_JOBS=$CPU_COUNT
+export NINJAFLAGS="-j${MAKE_JOBS}"
```

Comment:
is this line necessary? Is this the new best practice?

Reply:
qtwebengine uses ninja as build tool. @isuruf and I observed that building qtwebengine still uses all the cores by default. This line should avoid this from happening.